core/citrus-api/src/test/java/org/citrusframework/spi/ClassPathResourceResolverTest.java

```diff
@@ -8,20 +8,32 @@
 import java.util.Collections;
 import java.util.Enumeration;
 import java.util.List;
+import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Stream;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mockito;
 
 class ClassPathResourceResolverTest {
 
-    @Test
-    void loadFromFatJar() throws IOException {
+    private final ClasspathResourceResolver fixture = new ClasspathResourceResolver();
```

Comment:
as far as I remember, this should be a TestNG test. I think JUnit tests won't be executed when running via Maven. TestNG also has a concept very similar to parameterized tests, they call it data providers. have a look at methods with the annotation `@Test(dataProvider="...")` for examples.